### ISSUE TITLE
422 interception removes field names

### ISSUE THREAD

**Issue by Noogic** [mattias-persson/vue-axios-interceptors#3]
```
window.intercepted.$on('response:422', data => {
            console.log(data);
        });
```

In Laravel, when returning a `422` status, the data is an object with the errors associated to the field. That makes really easy to show each error under the wrong field, but the interceptors of this packages are removing the field name, so it's not possible to know to which input belongs each error.

**Comment by mattias-sanfridsson** [comment]
Valid point! It'd be better to parse them into an key-value object, as it would make it easy both to loop through them all and to reference a single field. I'll make sure to update this as soon as I can. Thanks for your input!

**Comment by mattias-sanfridsson** [comment]
Fixed in 0.3.0. You can now do this: 
```
window.intercepted.$on('response:422', data => {
    console.log(data.body.title); // Outputs the error for the title field.
});
```

Thanks for reporting!
